packages/wonder-blocks-form/components/choice-field.js

```diff
@@ -1,47 +1,74 @@
 
 type Props = {|
-    /** User-defined. Label for the field. */ label: string,
-    /** User-defined. Optional description for the field. */ description?: string,
-    /** User-defined. Whether this choice is checked. */ checked: boolean,
-    /** User-defined. Whether this choice option is disabled. Default false. */ disabled: boolean,
-    /** User-defined. Whether this choice is in error mode. Default false. */ error: boolean,
-    /** User-defined. Returns the new checked state of the component. */ onChange: (
-        newCheckedState: boolean,
-    ) => void,
-    /** User-defined. Takes either "radio" or "checkbox" value. */ variant:
-        | "radio"
-        | "checkbox",
+    /** User-defined. Label for the field. */
+    label: string,
+
+    /** User-defined. Optional description for the field. */
+    description?: string,
+
+    /** User-defined. Whether this choice is checked. */
+    checked: boolean,
+
+    /** User-defined. Whether this choice option is disabled. Default false. */
+    disabled: boolean,
+
+    /** User-defined. Whether this choice is in error mode. Default false. */
+    error: boolean,
+
+    /** User-defined. Returns the new checked state of the component. */
+    onChange: (newCheckedState: boolean) => void,
+
     /**
      * User-defined. Used for accessibility purposes, where the label id should
      * match the input id.
-     */ id?: string,
-    /** User-defined. Optional id for testing purposes. */ testId?: string,
-    /** User-defined. Optional additional styling. */ style?: any,
+     */
+    id?: string,
+
+    /** User-defined. Optional id for testing purposes. */
+    testId?: string,
+
+    /** User-defined. Optional additional styling. */
+    style?: any,
+
     /**
      * Ignored because only applicable to Choice components in a group.
      * @ignore
-     */ groupName?: string,
+     */
+    groupName?: string,
+
     /**
      * Ignored because only applicable to Choice components in a group.
      * @ignore
-     */ value?: string,
+     */
+    value?: string,
 |};
 
+/**
+ * ChoiceField is a checkbox with a label and optional description.
+ *
+ * Currently, ChoiceField only supports a checkbox. The radio variant is not
+ * available because radio buttons don't really exist by themselves out in the
+ * wild.
+ *
+ * ChoiceField is to be used independently of CheckboxGroup, RadioGroup, or
+ * other ChoiceField components. If you wish to use choices as part of a
+ * group, please look at CheckboxGroup and RadioGroup, which take the Choice
+ * component as children and auto-populate many props.
+ *
+ * This wrapper exists to allow for more explicit Flow typing.
+ */
 export default class ChoiceField extends React.Component<Props> {
     static defaultProps = {
         disabled: false,
         error: false,
     };
 
     render() {
-        return <ChoiceInternal {...this.props} />;
+        return <ChoiceInternal variant="checkbox" {...this.props} />;
```

Comment:
Hmm - I think it would be better to make it in `defaultProps` as then we could see it in StyleGuidist as being the default option! (Plus Flow would also then know that the prop isn't required, even though it looks like it is.)